### DISCUSSION TITLE
Few questions

### DISCUSSION THREAD

**Discussion by webNeat** [geeksblabla/blanat#35] [Q&A] [answered]
Hello,

Thank you for organizing this challenge, I like it :D

I have some questions:

1. What are the compilers/runtimes used for the languages: C++, Rust and Javascript? which versions?
2. Are we allowed to create multiple threads/processes (other then the ones created by the runtime)? I am thinking of worker_threads in Nodejs or tokio threads in Rust as examples.
3. What is the memory usage limit?
4. After submitting the PR and being added to the leaderboard, can we submit following PRs to improve the solution?

Thanks

**Comment by aboullaite** [comment, marked as answer]
Hello 👋🏽 

1. node `v18.19.0`, rustc `1.75.0`, cpp `12.2.0`
2. Yes you can
3. ~20GB
4. Yes you can
